dataikuapi/dss/admin.py

```diff
@@ -752,9 +752,9 @@ class DSSGlobalApiKey(object):
     """
     A global API key on the DSS instance
     """
-    def __init__(self, client, key):
+    def __init__(self, client, id):
```

Comment:
We are breaking the existing calls to `dataikuapi.dssclient.DSSClient.get_global_api_key` if we simply do that. 
Cf. this [comment](https://github.com/dataiku/dataiku-api-client-python/pull/167/files#r696773706) on the initial PR (cc @FChataigner ).

And you indeed don't want to call the `/admin/globalAPIKeys` endpoints, we should add another call to resolve the id from the key or keep the previous code with deprecated methods with the key.

Reply:
good catch, I forgot about this retro-compatibility.
I will add another method.

Not really keen to `resolve the id from the key` without deprecating the method as well, as it will imply we support searching for a key using the secret.
This is not ideal and the implementation would be ugly: search feature in REST API are usually design like `/resource/?attributeA=foo` but that means the key will be as a GET parameter (which is bad security wise for secrets). Therefore I should do a POST but I don't really find that approach very sexy, as this would collision with the POST for create.

Anyway you will see what I suggest in my next commit, probably the best compromise at the end.